<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#F2E4EE;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 179, 245);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="left_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="imagem_ederson">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>441</width>
         <height>231</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../../../../../Pictures/vou_resolver_isso kawaii.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="main_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="top">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>10</y>
            <width>231</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;TAKE CUTE&quot;;</string>
          </property>
          <property name="text">
           <string>Cadastrar Usu√°rio</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="main">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QFrame" name="cadastro">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLineEdit" name="inputNome">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>20</y>
               <width>181</width>
               <height>20</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">.QLineEdit{ border:none;
	background-color: #CACACA;
}</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>47</width>
               <height>13</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">.QLabel{
	font: 87 10pt &quot;Source Code Pro Black&quot;;
}</string>
             </property>
             <property name="text">
              <string>Nome</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="inputTelefone">
             <property name="geometry">
              <rect>
               <x>240</x>
               <y>20</y>
               <width>171</width>
               <height>20</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">.QLineEdit{ border:none;
	background-color: #CACACA;
}</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>240</x>
               <y>0</y>
               <width>71</width>
               <height>16</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">.QLabel{
	font: 87 10pt &quot;Source Code Pro Black&quot;;
}</string>
             </property>
             <property name="text">
              <string>telefone</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>47</width>
               <height>13</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">.QLabel{
	font: 87 10pt &quot;Source Code Pro Black&quot;;
}</string>
             </property>
             <property name="text">
              <string>Email</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="inputEmail">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>20</y>
               <width>181</width>
               <height>20</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">.QLineEdit{ border:none;
	background-color: #CACACA;
}</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>47</width>
               <height>13</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">.QLabel{
	font: 87 10pt &quot;Source Code Pro Black&quot;;
}</string>
             </property>
             <property name="text">
              <string>CPF</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="inputCpf">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>20</y>
               <width>181</width>
               <height>20</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">.QLineEdit{ border:none;
	background-color: #CACACA;
}</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_cadastrar">
             <property name="geometry">
              <rect>
               <x>260</x>
               <y>10</y>
               <width>141</width>
               <height>21</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: pink;
    font: bold 10px ;
	text-align:center;

    }</string>
             </property>
             <property name="text">
              <string>cadastrar</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="footer">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
